pom.xml

```diff
@@ -36,7 +36,7 @@
 		<spring-cloud-deployer-local.version>2.8.4-SNAPSHOT</spring-cloud-deployer-local.version>
 		<spring-cloud-deployer-cloudfoundry.version>2.8.4-SNAPSHOT</spring-cloud-deployer-cloudfoundry.version>
 
-		<spring-cloud-deployer-kubernetes.version>2.8.4-SNAPSHOT</spring-cloud-deployer-kubernetes.version>
+		<spring-cloud-deployer-kubernetes.version>2.9.0-SNAPSHOT</spring-cloud-deployer-kubernetes.version>
```

Comment:
In SCDF `2.11.0` the version of Dataflow, Skipper, and Deployers all line up to `2.11.x`. 
Prior to that, the versions were as follows:
- Dataflow 2.10.x 
- Skipper 2.9.x
- Deployer 2.8.x

So this change would not be possible w/o breakage.